### PR TITLE
[Issue 3855] Support swap TAO on SimpleSwap

### PR DESCRIPTION
**Pull request by tunghp2002**

### CODE REVIEW

**Comment by saltict** [comment]
🚀 Deployed on https://pr-3871--sw-web-runner.netlify.app

File: packages/extension-base/src/services/swap-service/handler/simpleswap-handler.ts [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+      return Promise.resolve([submitStep, params.selectedQuote.feeInfo]);
+    }
+
+    return Promise.resolve(undefined);
+  }
+
+  public async validateSwapRequest (request: SwapRequest): Promise<SwapEarlyValidation> {
+    try {
+      const fromAsset = this.chainService.getAssetBySlug(request.pair.from);
```
Comment (nampc1) [inline packages/extension-base/src/services/swap-service/handler/simpleswap-handler.ts:180]: also need to validate token info and swap pair key on SimpleSwap

**Review by nampc1 — CHANGES_REQUESTED** [review]

**Review by nampc1 — CHANGES_REQUESTED** [review]

**Review by nampc1 — APPROVED** [review]

File: packages/extension-koni-ui/src/Popup/Transaction/variants/Swap.tsx [hunk 2]
```diff
@@ -369,7 +377,14 @@ const Component = ({ targetAccountProxy }: ComponentProps) => {
...
     setFeeOptions(quote.feeInfo.feeOptions);
     setCurrentFeeOption(quote.feeInfo.feeOptions?.[0]);
-  }, []);
+
+    const currentRequest = {
+      ...currentQuoteRequest as SwapRequest,
+      currentQuote: quote.provider
+    };
+
+    setCurrentQuoteRequest(currentRequest);
```
Comment (S2kael) [inline packages/extension-koni-ui/src/Popup/Transaction/variants/Swap.tsx:386]: use this code instead

```typescript
    setCurrentQuoteRequest((oldRequest) => {
      if (!oldRequest) {
        return undefined
      }
      
      return {
        ...oldRequest,
        currentQuote: quote.provider
      };
    })
```

File: packages/extension-web-ui/src/Popup/Transaction/variants/Swap.tsx [hunk 3]
```diff
@@ -685,17 +709,35 @@ const Component = () => {
...
+
+      if (providerId && [SwapProviderId.CHAIN_FLIP_MAINNET, SwapProviderId.CHAIN_FLIP_TESTNET].includes(providerId)) {
+        return new BigN(CHAINFLIP_SLIPPAGE);
+      }
+
+      if (providerId && SwapProviderId.SIMPLE_SWAP.includes(providerId)) {
+        return new BigN(SIMPLE_SWAP_SLIPPAGE);
+      }
+
+      return currentSlippage.slippage;
```
Comment (S2kael) [inline packages/extension-web-ui/src/Popup/Transaction/variants/Swap.tsx:723]: Make a simple logic

**Pull request merged**